libafl/src/feedbacks/list.rs

```
@@ -83,7 +83,10 @@ where
         OT: MatchName,
         S: HasNamedMetadata,
     {
-        let observer = observers.get(&self.observer_handle).unwrap();
+        let Some(observer) = observers.get(&self.observer_handle) else {
```

This should return `Err`, not (semi-)silently return